src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -221,10 +221,15 @@
         [components/amount-selector {:disabled?     modal?
                                      :error         (or amount-error
                                                         (when-not sufficient-funds? (i18n/label :t/wallet-insufficient-funds)))
-                                     :input-options {:default-value  (str (money/to-fixed (money/wei->ether amount)))
-                                                     :max-length     21
-                                                     :on-focus       (fn [] (when (and scroll @scroll) (utils/set-timeout #(.scrollToEnd @scroll) 100)))
-                                                     :on-change-text (update-amount-fn timeout)}}]
+                                     ;; While a user is typing something like "0.00000..." it is a very bad idea to replace it with just "0"
+                                     ;; we might break his intent. Also that keeps the text-input from interfering when a user tries to
+                                     ;; fix his typo.
+                                     :input-options (merge
+                                                     (if (pos? amount)
```

Comment:
should be `(when (pos? amount)`

Reply:
```clj
(cond-> static-map
   (pos? amount) (assoc :default-value (str (money/t...))))
```
Would be good as well, I don't particularly like merging to introduce just one kv pair into map, but that's mostly matter of taste :)